config/locales/en.yml

```diff
@@ -835,7 +835,15 @@ en:
       Condition:                Condition
       CustomButton:             Button
       CustomButtonSet:          Buttons Group
+      Container:                Container
       ContainerGroup:           Pod
+      ContainerImageRegistry:   Image Registry
+      ContainerImage:           Image
+      ContainerNode:            Node
```

Comment:
This means that for example when adding a new alert, you get a choice to base it on `Host / Node` *or* `Node`.

Which is confusing because there is no mention of containers with the second node.

Was this really intentional? (Since I see you had to work around a similar problem for the menus..

Reply:
Well since these aliases are used everywhere its hard to tell if theres anywhere itll mess up :X

Maybe we can keep a list of shortened names and full names?
we can used the shortened names only in pages that are specifically in containers context  and full name when in use along side entities from other providers?